helm/prometheus-meta-operator/templates/prometheus-rules/app.rules.yml

```diff
@@ -15,7 +15,7 @@ spec:
       annotations:
         description: '{{`Management Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
         opsrecipe: app-failed/
-      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*"}
+      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",app=~"prometheus-meta-operator.*", team=~"batman|halo"}
```

Comment:
Just a question, this expression will only match the metric with label `team` set to either `batman` or `halo`, does this exclude any other instances? If so should they be covered?

Reply:
This will exclude other instances and I am not sure if they should be covered as we have deployment not satisfied. As Batman has an issue for this, I asked for their opinion: https://github.com/giantswarm/giantswarm/issues/14846

Reply:
Technically, we could split them yes.

Reply:
Do you want me to do it here?

Reply:
I did split the alert per team. I think it should be best if batman actually takes care of setting the correct team labels here

Reply:
https://github.com/giantswarm/app-exporter/pull/66

Reply:
Nice one, yeah this keeps us going in the meantime, thanks!